set.mm

```diff
@@ -171874,19 +172004,53 @@ seq m ( + , ( n e. ZZ |-> if ( n e. A , [_ n / k ]_ B , 0 ) ) ) ~~> x ) \/
   $}
 
   ${
-    $d A k x $.  $d B x $.  $d k x z $.
+    $d A k x $.  $d B x $.  $d Z k x $.
     $( A finite sum with an additional integer summand is an integer.
```

Comment:
What about "A finite sum all of whose summands is an integer is itself an integer (case where the summation set is the union of a finite set and a singleton)." ?

Reply:
OK, I will adapt the comment correspondingly.